spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/s3/S3CrtAsyncClientAutoConfigurationTests.java

```diff
@@ -77,6 +81,17 @@ void withCustomGlobalEndpointAndS3Endpoint() {
 		}
 	}
 
+	@Test
+	void withPathStyleAccessEnabled() {
+		contextRunner.withPropertyValues().run(context -> {
+			S3AsyncClient client = context.getBean(S3AsyncClient.class);
+			S3AsyncClient delegate = (S3AsyncClient) ReflectionTestUtils.getField(client, "delegate");
+			SdkClientConfiguration clientConfiguration = (SdkClientConfiguration) ReflectionTestUtils.getField(delegate, "clientConfiguration");
```

Comment:
If anybody can think of a way to accomplish this without the double reflection field access, I would love to do this more eloquently!